addons/mail/static/src/components/message/message.xml

```diff
@@ -180,6 +180,9 @@
                             </ul>
                         </t>
                     </div>
+                    <t t-if="message.subject and threadViewer and threadViewer.thread and (threadViewer.thread.id === 'inbox' or threadViewer.thread.id === 'history')">
```

Comment:
```suggestion
                    <t t-if="message.subject and threadViewer and [env.messaging.inbox, env.messaging.history].includes(threadViewer.thread)">
```
`includes` not especially necessary, but the idea would be to compare to the singleton instead of leaking the special ids.